Workbooks/WorkbooksMetadata.json

```diff
@@ -514,7 +514,7 @@
     "dataTypesDependencies": [ "CommonSecurityLog", "SecurityEvent", "Syslog" ],
     "dataConnectorsDependencies": [],
     "previewImagesFileNames": [ "AzureSentinelSecurityStatusBlack.png", "AzureSentinelSecurityStatusWhite.png" ],
-    "version": "1.2",
+    "version": "1.3",
```

Comment:
Why did you update this version I can't see changes in the SecurityStatus workbook json itself

Reply:
i was trying to fix the conflict that was created by other accepted PRs.  I have reverted this commit